src/components/ProjectCard/index.jsx

```diff
@@ -49,30 +50,49 @@ export default class ProjectCard extends Component {
     }
   };
 
+  handleCardClick = event => {
+    const {
+      project: { fileName },
+    } = this.props;
+
+    if (event.type === 'keypress') {
+      const code = event.charCode || event.keyCode;
+
+      if (code !== 32 && code !== 13) {
+        return;
+      }
+    }
+
+    this.props.history.push(`/projects/${fileName}`);
+  };
+
```

Comment:
I would stay away from this if we can handle it using the `Link` component.

Reply:
ok! will try it